rust/operator-binary/src/superset_db_controller.rs

```diff
@@ -311,16 +313,32 @@ fn build_init_job(
         .add_env_var_from_secret("ADMIN_PASSWORD", secret, "adminUser.password")
         .add_volume_mount(CONFIG_VOLUME_NAME, CONFIG_DIR)
         .add_volume_mount(LOG_CONFIG_VOLUME_NAME, LOG_CONFIG_DIR)
-        .add_volume_mount(LOG_VOLUME_NAME, LOG_DIR);
+        .add_volume_mount(LOG_VOLUME_NAME, LOG_DIR)
+        .resources(
+            ResourceRequirementsBuilder::new()
+                .with_cpu_request("100m")
+                .with_cpu_limit("500m")
+                .with_memory_request("128Mi")
+                .with_memory_limit("512Mi")
+                .build(),
+        );
```

Comment:
```suggestion
        .resources(
            ResourceRequirementsBuilder::new()
                .with_cpu_request("100m")
                .with_cpu_limit("400m")
                .with_memory_request("128Mi")
                .with_memory_limit("128Mi")
                .build(),
        );
```

Reply:
Missed both these, whoops. Updated in 87a973d